spec/support/factory_helper.rb

```diff
@@ -1,12 +1,11 @@
 # I'm currently using Fabrication gem to generate objects but if this changes now I can make the change in one place
```

Comment:
Interesting. I didn't like the abstraction as I read the code...but this makes total sense. Nice job.